sdk/cognitiveservices/Vision.Face/src/Generated/Models/PersonGroup.cs

```diff
@@ -30,16 +30,16 @@ public PersonGroup()
         /// <summary>
         /// Initializes a new instance of the PersonGroup class.
         /// </summary>
-        /// <param name="personGroupId">PersonGroupId of the target person
-        /// group.</param>
         /// <param name="name">User defined name, maximum length is
         /// 128.</param>
+        /// <param name="personGroupId">PersonGroupId of the target person
+        /// group.</param>
         /// <param name="userData">User specified data. Length should not
         /// exceed 16KB.</param>
         /// <param name="recognitionModel">Possible values include:
         /// 'recognition_01', 'recognition_02', 'recognition_03',
         /// 'recognition_04'</param>
-        public PersonGroup(string personGroupId, string name = default(string), string userData = default(string), string recognitionModel = default(string))
```

Comment:
should the order be preserved? this feels inconveniently silent for people who upgrades